extensions/wrapper/wrapper-common-mappers/src/test/java/de/sovity/edc/ext/wrapper/api/common/mappers/utils/UiAssetMapperTest.java

```diff
@@ -0,0 +1,64 @@
+package de.sovity.edc.ext.wrapper.api.common.mappers.utils;
+
+import de.sovity.edc.ext.wrapper.api.common.model.UiAssetCreateRequest;
+import de.sovity.edc.utils.JsonUtils;
+import de.sovity.edc.utils.jsonld.vocab.Prop;
+import lombok.val;
+import org.junit.jupiter.api.Test;
+
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.json;
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.value;
+import static org.mockito.Mockito.mock;
+
+class UiAssetMapperTest {
+    @Test
+    void test_buildAssetJsonLdContainsCustomJson() {
```

Comment:
Very nice that you created this test class and tested the other direction.

I think the private custom json properties are missing though?